jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -61,12 +113,20 @@ def __init__(self, *args, **kwargs):
         self.hub.subscribe(self, MarkersChangedMessage,
                            handler=self._on_markers_changed)
 
+        self.hub.subscribe(self, ChangeRefDataMessage,
+                           handler=self._on_refdata_change)
+
     @property
     def user_api(self):
-        return PluginUserApi(self, expose=('link_type', 'wcs_use_affine'))
+        return PluginUserApi(
+            self,
+            expose=(
+                'link_type', 'wcs_use_affine', 'viewer', 'layer'

```

Comment:
eventually we might want to include some of the preset methods, but we can leave them out of the public API for now if you think we might want to iterate on them further.

Reply:
Agreed. I'll add `'rotation_angle', 'east_left'` now, since those are likely the most useful and least likely to change.